tap_asana/streams/base.py

```diff
@@ -63,6 +67,10 @@ def invalid_token_handler(details):
 
 
 def asana_error_handling(fnc):
+    @backoff.on_exception(backoff.expo,
+                          requests.Timeout,
+                          max_tries=MAX_RETRIES,
+                          factor=2)
```

Comment:
Please define the constant for the factor as I can see so many backoffs here.

Reply:
Added a global variable for factor.